### PR TITLE
Update millennium contract with template

### PR DESCRIPTION
**Pull request by daliboz**
Update millennium contract overview, apply template and add auth accepted

### CODE REVIEW

File: content/millennium/dstu2/entities/contract.md [hunk 1]
```diff
@@ -9,7 +9,19 @@ title: Contract | DSTU 2 API
 
 ## Overview
 
-The Contract resource usage is limited to representing persons that are authorized to view patient data. The subject of the contract is the patient that is the target of this authorization. The actors on the contract are the related people that are authorized to view data. An example would be where a contract represents that a mother is authorized to view her children's data. The contract is that authorization in FHIR.
+The Contract resource is currently limited to representing people who are authorized to view a patient's data. For this reason, contract type, subtype, action and reason are always set to values related to consent for EHR sharing. The subject of the contract is the patient. The actors on the contract are the related people who are authorized to view the subject patient's data. An example is a contract authorizing a mother (actor) to view her child’s data (subject).
```
Comment (MaxPhilips) [inline content/millennium/dstu2/entities/contract.md]: The last apostrophe in this paragraph is a curved quote, but it seems to render the same on the site as the straight quotes

File: content/millennium/dstu2/entities/contract.md [hunk 2]
```diff
@@ -39,28 +56,69 @@ Notes:
 - The `_include` parameter may be provided once for each resource to be included. Example: `_include=Contract:actor&_include=Contract:patient`
 - Additional OAuth2 scopes are required when the `_include` parameter is provided. To include with `Contract:actor` or `Contract:patient` values, the OAuth2 token must include scopes for one or both RelatedPerson.read and Patient.read, respectively, depending on the combination of resources to be included.
 
-### Response
+### Headers
+
+[//]: # (Required. Add all the required headers for the request, if anything besides accept and auth headers)
```
Comment (MaxPhilips) [inline content/millennium/dstu2/entities/contract.md]: Remove

**Comment by MaxPhilips** [comment]
+1

**Comment by dennispatterson** [comment]
+1

**Pull request merged**